src/Build/Graph/ProjectInterpretation.cs

```diff
@@ -35,6 +35,7 @@ internal sealed class ProjectInterpretation
         private const string PlatformMetadataName = "Platform";
         private const string PlatformsMetadataName = "Platforms";
         private const string EnableDynamicPlatformResolutionPropertyName = "EnableDynamicPlatformResolution";
+        private const string EnableDynamicPlatformFilteringPropertyName = "EnableDynamicPlatformFiltering";
```

Comment:
This should be mentioned in https://github.com/dotnet/msbuild/blob/main/documentation/ProjectReference-Protocol.md